### PR TITLE
Documentation for AIX 7.3 added

### PR DESCRIPTION
**Pull request by aklyachkin**
Added description how to configure AIX 7.3 to be IPA client.

### CODE REVIEW

File: src/page/ConfiguringAixClients.rst [hunk 1]
```diff
@@ -7,26 +7,367 @@ Guide <FreeIPAv1:Client_Configuration_Guide>`__
...
+| sudo_ids RPM package                             | `AIX Toolbox for Open source applications <https://www.ibm.com/support/pages/aix-toolbox-open-source-software-downloads-alpha#S>`__ |
++--------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
+
+You must also obtain several parameters from your FreeIPA installation before proceeding further.
+
+* FreeIPA server names and their IP addresses
+* FreeIPA DNS domain
+* FreeIPA Kerberos realm
+* FreeIPA LDAP base DN
+* HBAC rule ID for access to your AIX server (:command:`ipa hbacrule-show <name> --all | grep dn:`)
```
[the remark this answers is not in the document]
Comment (aklyachkin) [reply src/page/ConfiguringAixClients.rst:39]: No, it is not mandatory. But this is a standard FreeIPA feature, nothing special to AIX. E.g. see https://www.freeipa.org/page/Howto/HBAC_and_allow_all and https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_idm_users_groups_hosts_and_access_control_rules/configuring-host-based-access-control-rules_managing-users-groups-hosts

That's why there is no such example. 

Another curious thing I found out, that this page is linked only through FreeIPA V1 Documentation which is not shown anywhere on the doc page. Should I add a link to it on the Howtos https://www.freeipa.org/page/HowTos page?

**Comment by aklyachkin** [comment]
I did some changes:
- added note to HBAC and linke to HBAC_and_allow_all document
- added link from HowTos to ConfiguringAixClients
- added some troubleshooting information
- reformated text a little bit and corrected some typos

**Comment by rcritten** [comment]
I have no way to validate the content but this looks ok to me.

**Comment by aklyachkin** [comment]
Thank you Rob!

**Pull request merged**